src/frontend/staff-portal/src/app/components/staff-workbench/ticket-inbox/ticket-inbox.component.ts

```diff
@@ -132,64 +131,63 @@ export class TicketInboxComponent implements OnInit, AfterViewInit {
+        if (this.getOcrViolationErrors(fields["counts.count_no_2.is_regulation"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_2.section"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_2.ticketed_amount"])) { return true; }
 
         // count 3
-        if (this.getOcrViolationErrors(fields["counts.count_no_3.description"])) { return this.SystemDetectedOcrIssues.Y; }
-        if (this.getOcrViolationErrors(fields["counts.count_no_3.act_or_regulation_name_code"])) { return this.SystemDetectedOcrIssues.Y; }
-        if (this.getOcrViolationErrors(fields["counts.count_no_3.is_act"])) { return this.SystemDetectedOcrIssues.Y; }
-        if (this.getOcrViolationErrors(fields["counts.count_no_3.is_regulation"])) { return this.SystemDetectedOcrIssues.Y; }
-        if (this.getOcrViolationErrors(fields["counts.count_no_3.section"])) { return this.SystemDetectedOcrIssues.Y; }
-        if (this.getOcrViolationErrors(fields["counts.count_no_3.ticketed_amount"])) { return this.SystemDetectedOcrIssues.Y; }
-
-        if (this.getOcrViolationErrors(fields.court_location)) { return this.SystemDetectedOcrIssues.Y; }
-        if (this.getOcrViolationErrors(fields.detachment_location)) { return this.SystemDetectedOcrIssues.Y; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_3.description"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_3.act_or_regulation_name_code"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_3.is_act"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_3.is_regulation"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_3.section"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_3.ticketed_amount"])) { return true; }
+
+        if (this.getOcrViolationErrors(fields.court_location)) { return true; }
+        if (this.getOcrViolationErrors(fields.detachment_location)) { return true; }
+        return false;
       }
+      else return null;
     } catch (e) {
-      return this.SystemDetectedOcrIssues.Y;
+      return null;
     }
-
-    // step through fields in deserialized object and look for validation errors
-    return this.SystemDetectedOcrIssues.N;
   }
 
   // return number of validation errors
-  getOcrViolationErrors(field?: Field): DisputeSystemDetectedOcrIssues {
-    if (field == undefined || field == null) return this.SystemDetectedOcrIssues.N;
-    if (field?.fieldConfidence && field.fieldConfidence < 0.8) {
-      return this.SystemDetectedOcrIssues.Y;
-    } else return this.SystemDetectedOcrIssues.N;
+  getOcrViolationErrors(field?: Field): boolean {
+    if (field == undefined || field == null) return false;
+    if (field?.fieldConfidence != null && field.fieldConfidence < 0.8) {
```

Comment:
Are we going to make the confidence comparison (0.8) an ENV var?

Reply:
It is possible to expose environment variables in nginx via a virtual configuration file. This was one of the things I wanted to wrap up the application version details used in the citizen portal. 

https://serverfault.com/questions/676680/can-i-use-nginx-environment-variables-within-static-files-that-nginx-serves

```
        location ~ ^/config.js$ {
            add_header Content-Type text/javascript;
            set_by_lua $env_var 'return os.getenv("MY_ENV_VAR")';
            return 200 'const MY_ENV_VAR = \'$env_var\'';
        }
```

Reply:
this use of the threshold in the staff portal just highlights items for VTC staff to look at.  Staff can still choose to validate the ticket regardless of whether this shows as true or not.  The threshold that was discussed during yesterday's meeting was the one for critical fields in the citizen portal that determines whether or not a citizen can proceed.